Bonsai.Configuration/Bonsai.Configuration.csproj

```diff
@@ -7,6 +7,9 @@
   <ItemGroup>
     <ProjectReference Include="..\Bonsai.NuGet\Bonsai.NuGet.csproj" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
I did try to have everything become .NET Standard at one point, even for the .NET Framework bootstrapper. The problem is that this ended up pulling literally dozens of dependencies to the bootstrapper and filling the `Packages` directory with essentially half of the .NET Framework assemblies (since .NET Standard cannot assume that any of these exist).

It could be that I was doing something wrong with package dependency resolution at the bootstrapper level, but things were much cleaner by just targeting .NET Framework. I will try again and if there is no difference now we can just do that.

Reply:
I’m in favor of just dual targeting during the transition period and avoiding .NET Standard altogether. I’m asking for this package specifically.

Maybe the better question is to ask why this package targets .NET Standard rather than .NET?

Reply:
I think this may have been related to the fact that both the bootstrapper and the player projects use this package as a dependency and were targeting different frameworks, somehow it felt simpler, but I think you are right and there is probably no need to target .NET Standard now.